components/BlogPost.tsx

```diff
@@ -0,0 +1,40 @@
+import Image from 'next/image';
+
+import styles from './BlogPost.module.css';
+
+type BlogPostProps = {
```

Comment:
This is fine, but usually, in our project, we tend to use interfaces over types.

The difference is very subtle, so this is not an issue at all!

Reply:
I was just thinking about that this morning, and I realized that I don’t really know the difference between those two, so I will look into it and change types for interfaces.